src/Symfony/Bundle/FrameworkBundle/Resources/config/cache_pools.xml

```diff
@@ -58,5 +39,20 @@
             <argument /> <!-- default lifetime -->
         </service>
 
+        <service id="cache.adapter.shared" alias="cache.adapter.filesystem" />
```

Comment:
I put the adapters here to follow the reading order (as in the configuration, the extension, etc.)